common/src/main/java/com/wynntils/services/mapdata/MapFeatureRenderer.java

```diff
@@ -71,16 +74,25 @@ public static void renderMapFeature(
                     0,
                     iconWidth,
                     iconHeight);
+            BufferedFontRenderer.getInstance()
```

Comment:
What happened here?

Reply:
Left behind debug code to render the zoom level. I've thought I force pushed to remove this.. Anyways, does this look acceptable otherwise?